src/entities/DealTokenSwap.ts

```diff
@@ -230,15 +230,15 @@ export class DealTokenSwap implements IDeal {
 
   @computedFrom("daoTokenClaims")
   public get isFullyClaimed(): boolean {
-    if (!this.daoTokenClaims) return false;
+    if (!this.isClaiming || !this.daoTokenClaims) return false;
```

Comment:
please return from functions in only one place.  (also prefer braces around if/else blocks.

Reply:
Done